Dockerfile

```diff
@@ -16,7 +16,7 @@
 #
 FROM alpine:latest as pre-build
 
-ARG APISIX_DASHBOARD_VERSION=v2.0
+ARG APISIX_DASHBOARD_VERSION=master
```

Comment:
Good catch! and should we still keep this file? We have an an-in-one Dockerfile here[1]. 🤔

cc @membphis @liuxiran @gxthrj @ShiningRush 

[1] https://github.com/apache/apisix-docker/blob/master/all-in-one/apisix-dashboard/Dockerfile

Reply:
> Good catch! and should we still keep this file? We have an an-in-one Dockerfile here[1]. 🤔
> 
> cc @membphis @liuxiran @gxthrj @ShiningRush
> 
> [1] https://github.com/apache/apisix-docker/blob/master/all-in-one/apisix-dashboard/Dockerfile

@juzhiyuan that's right. In any case, we'd better not keep two copies for a single file. The `all-in-one` is better for me (of course, we also need this patch for that). While, I would say there's one question that's worth noting: how can we build a docker image for apisix-dashboard continuously once every single commit gets merged, if using a dockerfile stored in another repository? Looking forward to your insights. Thanks.

Reply:
Oh yes, I'm working on Dashboard Image in DockerHub, once it's approved from ASF, we could add CI to auto-publish Docker Image to DockerHub.

Reply:
> Oh yes, I'm working on Dashboard Image in DockerHub, once it's approved from ASF, we could add CI to auto-publish Docker Image to DockerHub.

@juzhiyuan that's great, and I would choose the  `all-in-one`. Let's see others' opinions. Thank you very much.

Reply:
We could merge this one first :) When that repo is created, let's reupdate this repo's documentation.

Reply:
> We could merge this one first :) When that repo is created, let's reupdate this repo's documentation.

@juzhiyuan ok, thank you very much. Shall we need to fix the same issue at https://github.com/apache/apisix-docker/blob/master/all-in-one/apisix-dashboard/Dockerfile#L36 for now?

Reply:
Ya!

Reply:
> Ya!

@juzhiyuan ok, I'll fix that soon.